src/patternfly/components/Drawer/drawer.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-drawer {
+  --pf-c-drawer__main--MarginRight: calc(var(--pf-c-drawer__panel--Width) * -1);
+  --pf-c-drawer--m-open__main--MarginRight: 0;
+  --pf-c-drawer__main--Transition: margin .25s ease;
+  --pf-c-drawer__panel--ZIndex: var(--pf-global--ZIndex--sm);
+  --pf-c-drawer__panel--Width: 100%;
+  --pf-c-drawer__panel--md--Width: #{pf-size-prem(450px)};
+  --pf-c-drawer__panel--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-drawer__panel--Transition: transform .25s ease;
```

Comment:
Why do `--pf-c-drawer__main` and `--pf-c-drawer__panel` both have a transition?

Reply:
`__main` transitions when `pf-c-drawer` has `.pf-m-inline`. I can limit the `transition` to when the drawer has that class if you think this will cause problems. Only problem I see is if someone dynamically changes the `margin-right` (I changed the transition from `margin` to `margin-right`) outside of the context of the drawer opening/closing, that margin change will be transitioned. Moving the transition to `.pf-m-inline` will still cause that, but only when the drawer also has `.pf-m-inline`

Reply:
I'm still a little confused here could you explain more as to why they both have a transition